astropy/io/votable/converters.py

```diff
@@ -251,7 +251,7 @@ def output(self, value, mask):
         """
         raise NotImplementedError("This datatype must implement a 'output' method.")
 
-    def binparse(self, read):
+    def binparse(self, read, config=None):
```

Comment:
`config` needs to be mentioned in the docstring. 

And it's a new addition, let's add it as mandatory kwarg, too, so we start passing it around explicitly.

Reply:
I changed it into this:

`
def binparse(self, read, *, config=None):
`

forcing the use as a kwarg. Did you also mean that I should remove the `=None` part, making it a mandatory kwarg?

Reply:
Nope, this shouldn't be made mandatory but have a reasonable default.